src/nvim/eval/funcs.c

```diff
@@ -3188,6 +3195,12 @@ static void f_getcmdpos(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   rettv->vval.v_number = get_cmdline_pos() + 1;
 }
 
+/// "getcmdscreenpos()" function
+static void f_getcmdscreenpos(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+{
+  rettv->vval.v_number = get_cmdline_screen_pos() + 1;
```

Comment:
what does this mean? how is position just a number? what happens when `ext_cmdline` is set?

Reply:
It means screen cursor position in command line.
It is needed for command line popup feature.

I don't test `ext_cmdline`.  But it should work.